### DISCUSSION TITLE
Duplicating `matchDatasources` - one with `groupName` and one without. Muliple PR's + grouped PR.

### DISCUSSION THREAD

**Discussion by bukowa** [renovatebot/renovate#29636] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

Hey I am really interested in below configuration; I would like to see 1 PR for all npm packages and multiple PR's - one per npm package. If automated tests pass for the grouped PR then I can merge it as once, probably customizing the commit message so automatic changelog generator can pick them up. Then if any package brakes my github action test's - I can clearly see in the PR which one is it.

Does it make sense? Is it possible or should it be?

````json
    {
      "matchDatasources": [
        "npm"
      ],
      "labels": [
        "javascript"
      ],
    },
    {
      "matchDatasources": [
        "npm"
      ],
      "groupName": "npm modules",
      "labels": [
        "javascript"
      ]
    }
````
What happens with this configuration: all npm packages are grouped into one PR. I would like to see per package PR too!

**Comment by rarkins** [comment]
You cannot have both grouping and separate

**Reply by bukowa** [reply]
Can it be a feature request? It sounds logical to make it work, what do you think?

**Reply by rarkins** [reply]
Similar conceptually to https://github.com/renovatebot/renovate/issues/4404
